packages/app/src/app/pages/Dashboard/Content/routes/Settings/components/PermissionSettings.tsx

```diff
@@ -118,8 +118,9 @@ const MinimumPrivacy = ({ disabled }: { disabled: boolean }) => {
       gap={114}
       css={css({
         padding: 6,
+        backgroundColor: '',
         border: '1px solid',
-        borderColor: 'grays.600',
+        borderColor: 'transparent',
```

Comment:
Do we still need the border? If not, we can remove it altogether. This will remove the `1px` space though. This is the case for the border changes in other files too I think.